src/App.js

```diff
@@ -1,8 +1,16 @@
-import './App.css';
+import React from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
```

Comment:
You successfully created a single page application (SPA) using react-router-dom to serve each page without reloading 🙌🔥

Reply:
Thank you :)